src/main/java/com/puppycrawl/tools/checkstyle/AbstractAutomaticBean.java

```diff
@@ -338,9 +341,10 @@ private static final class UriConverter implements Converter {
 
         @SuppressWarnings("unchecked")
         @Override
+        @Nonnull
         public Object convert(Class type, Object value) {
             final String url = value.toString();
-            URI result = null;
+            @Nullable URI result = null;
 
             if (!CommonUtil.isBlank(url)) {
                 try {
```

Comment:
This comment is for line 358: we may need to cast this value to make checker happy.